nucliadb/nucliadb/tests/benchmarks/test_search.py

```diff
@@ -69,6 +70,7 @@ async def test_search_returns_labels(
     warmup=False,
 )
 @pytest.mark.asyncio
+@pytest.mark.parametrize("knowledgebox", ["STABLE", "EXPERIMENTAL"], indirect=True)
```

Comment:
oh, nice way to parametrize a fixture! I had not been able to understand this from reading the pytest docs